setup/www/resources/config/ssl-defaults.conf

```diff
@@ -0,0 +1,17 @@
+ssl_certificate ssl/nodejs_chained.crt;
+ssl_certificate_key ssl/nodejs.key;
+ssl_trusted_certificate ssl/nodejs_chained.crt;
+ssl_dhparam ssl/dhparam.pem;
+
+ssl_ciphers ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS;
+ssl_prefer_server_ciphers on;
+ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+
+ssl_session_cache shared:SSL:10m;
+ssl_session_timeout 24h;
```

Comment:
Should we ad  `ssl_session_tickets off;` ([source](https://mozilla.github.io/server-side-tls/ssl-config-generator/?server=nginx-1.9.5&openssl=1.0.1e&hsts=yes&profile=intermediate))